### PR TITLE
[SQL-DS-CACHE-24][POAE7-893] some code improvement for APE cache

### PR DESCRIPTION
**Pull request by iyupeng**
## What changes were proposed in this pull request?

* update .gitignore
* update some pom.xml files
* make flink's native wrapper configurable on whether to use cache or not
* fix hostname missing in cache location

## How was this patch tested?

integration tests,

### CODE REVIEW

File: oap-ape/ape-java/ape-flink/src/main/java/org/apache/flink/formats/parquet/utils/ParquetNativeRecordReaderWrapper.java [hunk 1]
```diff
@@ -99,15 +100,32 @@ public long initialize(Configuration hadoopConfig, RowType projectedType, FileSo
...
+        }
+
+        LOG.info("native parquet reader initialized, plasma cache: {}, cache locality: {}",
+                plasmaCacheEnabled, cacheLocalityEnabled);
 
         return reader;
 
     }
 
+    private void setRedisToNativeReader(Configuration hadoopConfig) {
```
Comment (winningsix) [inline oap-ape/ape-java/ape-flink/src/main/java/org/apache/flink/formats/parquet/utils/ParquetNativeRecordReaderWrapper.java]: Let's change this into 1 line code in L109.
```ParquetReaderJNI.setPlasmaCacheRedis(reader, hadoopConfig.get(Constants.CONF_KEY_FS_APE_HCFS_REDIS_HOST, Constants.DEFAULT_REDIS_HOST)
, hadoopConfig.getInt(Constants.CONF_KEY_FS_APE_HCFS_REDIS_PORT, Constants.DEFAULT_REDIS_PORT)
, hadoopConfig.get(Constants.CONF_KEY_FS_APE_HCFS_REDIS_AUTH, Constants.DEFAULT_REDIS_AUTH)
);```

File: oap-ape/ape-java/ape-flink/pom.xml [hunk 2]
```diff
@@ -58,6 +58,12 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>com.intel.oap</groupId>
+      <artifactId>ape-hcfs</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
```
Comment (winningsix) [inline oap-ape/ape-java/ape-flink/pom.xml]: Let's align on versions for all modules. Current is 1.1.0-snapshot.

**Review by winningsix — COMMENTED** [review]
LGTM. A few minor comments.

**Comment by jikunshang** [comment]
LGTM.

**Pull request merged**